Dockerfile

```diff
@@ -14,6 +14,14 @@ RUN npm install -g npm
 # install typescript compiler
 RUN npm install -g typescript
 
+# install git
+RUN apt update --no-install-recommends -yq \
+    && apt-get install git -yq
+#install Emscripten
+FROM emscripten/emsdk:1.38.25
+# Install required tools that are useful for this project i.e. music-blocks
+ RUN apt update && apt install -y music-blocks
```

Comment:
Could you explain this line?

Reply:
![image](https://user-images.githubusercontent.com/50985033/105632562-cc811f00-5e79-11eb-93fc-0a48309733bf.png)


It's a snippet from the official docker hub [page ](https://hub.docker.com/r/emscripten/emsdk) for emscripten.

Reply:
There is no `music-blocks` in the `APT` repository

Reply:
However, `FROM emscripten/emsdk:1.38.25` should fetch the `emscripten` layers. Pls check building the image and verify the `emcc` compilation of a `.c`/`.cpp` file to `.wasm`